capstone/src/main/java/com/google/sps/servlets/ClubServlet.java

```diff
@@ -1,20 +1,24 @@
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Map;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example club content*/
+/** Servlet that returns some example club content. */
 @WebServlet("/clubs")
-public class DataServlet extends HttpServlet {
-  private Map<String, Club> clubs = new HashMap<String, Club>();
-  
+public class ClubServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    setUp();
+    HardcodedClubs hc = new HardcodedClubs();
+    Map<String, Club> clubs = hc.clubs;
+    Gson gson = new Gson();
+    // Only accesses Club club 1 for now
+    String json = gson.toJson(clubs.get("club 1"));
```

Comment:
Be sure to edit the key here if you are changing the keys to capitalize "club", so here it would be "Club 1" instead